content/apps/govcloud.md

```diff
@@ -2,13 +2,13 @@
 menu:
   main:
     parent: apps
-title: What Is New in Govcloud?
+title: What's New in Cloud.gov?
```

Comment:
Heh, no, this is actually correct. cloud.gov is in the process of moving to GovCloud...confusing, I know.

https://aws.amazon.com/govcloud-us/